### PR TITLE
Frontend/feature/landing page

### PR DESCRIPTION
**Pull request by WachirapatMT**
**Changes**
-  Add logo to Sider
-  Add home option to Sider for all user types
-  Add user avatar image for signed-in users
-  Implement responsive landing page

**Results**
- Desktop
image
image

- Tablet (Ipad)
image
image

- Mobile (IphoneX)
image
image
image

### CODE REVIEW

File: frontend/components/Sider.js [hunk 1]
```diff
@@ -1,19 +1,56 @@
...
+  const [isBroken, setIsBroken] = useState(collapsed);
+  const [showLogoText, setShowLogoText] = useState(!collapsed);
+
+  useEffect(() => {
+    if (collapsed) {
+      setTimeout(() => setShowLogoText(!collapsed), 30)
+    } else {
+      setTimeout(() => setShowLogoText(!collapsed), 145)
+    }
+  }, [collapsed])
```
Comment (WachirapatMT) [inline frontend/components/Sider.js:19]: smoothing responsive logo on collapsing

File: frontend/components/Header.js [hunk 2]
```diff
@@ -49,12 +48,21 @@ const NonGuestHeader = ({ username, profileImageUrl, handleSignOut }) => {
...
 
-  // TODO: Set image to center
   return (
-    <div className="font-bold text-lg">
-      <UserOutlined className="text-lg mr-4 rounded-full bg-gray-200 h-10 w-10 inline-flex items-center justify-center cursor-pointer " />
-      {/* <Image className='rounded-full my-auto mx-0' src={profileImageUrl} width={40} height={40} /> */}
-      {username}
+    <div className="flex items-center text-lg">
+      {profileImageUrl ? (
+        <div className="rounded-full mt-1.5">
```
Comment (WachirapatMT) [inline frontend/components/Header.js:54]: use `<div/>` since `<Image />` cannot be styled directly

File: frontend/components/Sider.js (hunk 1 above)
Comment (KKCORR) [reply frontend/components/Sider.js:19]: In another way, we can use `collapsed` to toggle className like `hidden` and add a transition to it.

Comment (WachirapatMT) [reply frontend/components/Sider.js:19]: In this case, when collapse changes true->false and false->true, they use different delay time and need highly customized delay time. Transition delay might not be able to deal with it.

Comment (KKCORR) [reply frontend/components/Sider.js:19]: That's ok.

**Pull request merged**